src/Field/Array_.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+// vim:ts=4:sw=4:et:fdm=marker:fdl=0
+
+namespace atk4\data\Field;
+
+use atk4\data\Field;
+use atk4\data\ValidationException;
+
+/**
+ * Array field type. Think of it as field type "array" in past.
+ */
+class Array_ extends Field
+{
+    /** @var string Field type for backward compatibility. */
+    public $type = 'array';
+
+    /**
+     * Validate and normalize value.
+     *
+     * @param mixed $value
+     *
+     * @throws ValidationException
+     *
+     * @return mixed
+     */
+    public function normalize($value)
+    {
+        if ($value === null || $value === '') {
+            if ($this->required) {
+                throw new ValidationException([$this->name => 'Must not be null or empty']);
```

Comment:
shouldn't we call parent?

Reply:
I am avoiding calling parent normalize because was trying to get rid of it at first and then see if a lot of field classes will actually duplicate same code over and over again, then move it back to parent class normalize method.